README.md

```diff
@@ -1 +1,35 @@
-# chatty-instructors
\ No newline at end of file
+## Description
+A Narnia themed group project making a chatroom to practice modules, SASS styling, and bootstrap. 
+
+### Feature List
+* Multiple Users
+* Timestamps
+* Seed Data
+* Message Limit
+* Dark Theme
+* Large Text
+
+<!-- ## Screenshots
+![Main View](./images/pet-adoption.png) -->
+
+## How To Run
+1. Clone down this repo
+1. Use your favorite http server (like [http-server](https://www.npmjs.com/package/http-server)) to serve it up (`hs`)
+1. In your browser, go where it's being served (default is localhost:8081)
```

Comment:
`hs` is dead to us. I think our instructions need to say use `npm start` instead